services/rds/src/main/java/org/finra/gatekeeper/controllers/AccessRequestController.java

```diff
@@ -109,6 +109,11 @@ public Object cancelRequest(@RequestBody ActiveAccessRequestWrapper request) {
         return accessRequestService.cancelRequest(request.getTaskId(), request.getId());
     }
 
+    @RequestMapping(value = "/updateRequestApproverComment", method = RequestMethod.PUT, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I Know we're only going to be allowing the request approver comment at this point Can we keep the name of this more general? like "updateRequest"